packages/portal/src/index.html

```diff
@@ -12,6 +12,7 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
     />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
This should not be needed as you're using font awesome icons, not the material ones

Reply:
Yes, you can remove it, if its not need or do you want me to remove it and make a new PR ? @sinedied